python/CHANGELOG.rst

```diff
@@ -6,6 +6,12 @@ In development
 
 **Breaking changes**
 
+- Mutations now have a double-precision floating-point ``time`` column. This is
```

Comment:
Same comment as the C changelog.

Reply:
Fixed in https://github.com/tskit-dev/tskit/pull/672/commits/532f50baa5f427a5b8b80058ba00ca0a29f52f63